scalafix/src/main/scala/org/typelevel/sbt/TypelevelScalafixPlugin.scala

```diff
@@ -0,0 +1,30 @@
+package org.typelevel.sbt
+
+import sbt._
+import scalafix.sbt.ScalafixPlugin
+
+import Keys._
+import ScalafixPlugin.autoImport._
+
+object TypelevelScalafixPlugin extends AutoPlugin {
+
+  override def requires = ScalafixPlugin
+
+  override def trigger = allRequirements
+
+  object autoImport {
+    val tlScalafixDependencies = settingKey[Seq[ModuleID]]("The scalafix rule dependencies to enable in the build.")
+  }
+
+  import autoImport._
+
+  override def buildSettings = Seq[Setting[_]](
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision,
+    scalafixScalaBinaryVersion := CrossVersion.binaryScalaVersion(scalaVersion.value),
+    tlScalafixDependencies := Seq(
+      "com.github.liancheng" %% "organize-imports" % "0.6.0"
+    ),
+    scalafixDependencies ++= tlScalafixDependencies.value
```

Comment:
Is there a reason we need our own `tl` setting for this one? It seems reasonable to just append to the existing setting.

Reply:
No reason, that just seemed to be the convention in this plugin e.g. `tlMimaPreviousVersions` 😛 I will remove it

Reply:
Ah, that's because there is no such equivalent setting in MiMa itself :) see https://github.com/typelevel/sbt-typelevel/pull/176.